cmd/postgres_exporter/queries.go

```diff
@@ -142,12 +221,13 @@ var queryOverrides = map[string][]OverrideQuery{
 			(
 				SELECT
 					datname,
+					pid,
```

Comment:
as far as I remember this pid value creates high cardinality and brings a problem with bad performance and high disk space usage.